app/src/main/java/com/sdp13epfl2021/projmag/notification/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package com.sdp13epfl2021.projmag.notification
+
+class Constants {
+    companion object{
+        const val BASE_URL = "https://fcm.googleapis.com"
+        const val SERVER_KEY= "AAAAwNSBlt8:APA91bHNLxj_wt6HCQ9Lfk3_6sSYIYHxfSRA7qgR_yZP-zCYxpgvBHlttayDPn20zKundac4-3RmyrcNwJPZagYLLuXOVOKljgVLRmIEZbLYODAXT1RblGZlddwa4U-SmCkURBlVs6TC"
```

Comment:
# :warning:  **Attention** :warning: 

You should **never** post a **private** key on a public repository.
Anybody can use it to send notification and messages in our application, this is a serious security issue.

I suggest that you read this [documentation](https://github.com/sweng-epfl/public/blob/master/project/03-Operations/Handout.md#ignoring-files): basically using a `*.xml` file with our API keys in it, adding it to `.gitignore`, and use github secret to store the key.

Then we will certainly need to recreate a key, but we should meet with the team to be sure that we do not break anything.